src/renderer/components/+workloads-statefulsets/stateful-set-menu.tsx

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency